csf_tz/csf_tz/doctype/inter_company_stock_transfer/inter_company_stock_transfer.py

```diff
@@ -7,13 +7,12 @@
 
 class InterCompanyStockTransfer(Document):
     def before_submit(self,warehouse=None):
-        item_list_from = []
-        item_list_to = []
+        item_list_from, item_list_to = [], []
 
         for item in self.items_child:
             
-            valuation_rate = get_valuation_rate(item.item_code, self.from_company, self.default_from_warehouse)
-            if valuation_rate == 0 or valuation_rate is None:
+            valuation_rate_data = get_valuation_rate(item.item_code, self.from_company, self.default_from_warehouse)
+            if valuation_rate_data.valuation_rate == 0 or valuation_rate_data.valuation_rate is None:
```

Comment:
if not valuation_rate_data or valuation_rate_data.valuation_rate is None or valuation_rate_data.valuation_rate == 0: